package/ntrip_daemon/src/ntrip_settings.c

```diff
@@ -150,24 +151,57 @@ void ntrip_stop_processes()
   }
 }
 
+/* Move to libpiksi as generic function */
+static bool file_read_value(char *file_path) {
```

Comment:
Could un-static this function in libpiksi: https://github.com/swift-nav/piksi_buildroot/blob/84ee95fe09f9196c49fee0a0306cea23601f5683/package/libpiksi/libpiksi/src/util.c#L34